tests/sentry/db/models/fields/bitfield/test_bitfield.py

```diff
@@ -331,16 +338,12 @@ def test_dictionary_init(self):
         pytest.raises(ValueError, BitField, flags={"1": "non_int_key"})
 
     def test_defaults_as_key_names(self):
-        class TestModel(models.Model):
-            class Meta:
-                app_label = "fixtures"
-
-            flags = BitField(
-                flags=("FLAG_0", "FLAG_1", "FLAG_2", "FLAG_3"), default=("FLAG_1", "FLAG_2")
-            )
-
-        field = TestModel._meta.get_field("flags")
-        self.assertEqual(field.default, TestModel.flags.FLAG_1 | TestModel.flags.FLAG_2)
+        field = BitFieldTestModelWithDefaultsAsKeyNames._meta.get_field("flags")
```

Comment:
Reason for this change is it's not cleaning up stuff in the db properly for tests. Making the format similar to how the other test models are declared here

Reply:
Nice, so this is the reason we weren't able to make backup tests earlier, is that it?

Reply:
that and the integrity error from the duplicate key violation issue